### PR TITLE
Jm/hedgil hedging

### PR DESCRIPTION
**Pull request by jmonteer**

### CODE REVIEW

**Comment by jmonteer** [comment]
only first test running (need to adapt the rest of them)

File: contracts/Joint.sol [hunk 1]
```diff
@@ -149,11 +165,21 @@ abstract contract Joint {
...
         IERC20(address(pair)).approve(address(router), type(uint256).max);
+        IERC20(address(pair)).approve(address(masterchef), type(uint256).max);
+
+        // HEDGIL
+        // NOTE: tokenB will always be used as quoteToken (to pay hedges)
+        hedgil = IHedgilV1(_hedgil);
+        require(tokenB == address(hedgil.quoteToken()), "!hedgil setup");
+        IERC20(tokenB).approve(address(hedgil), type(uint256).max);
+
+        hedgeBudget = 100; // 0.5%
```
Comment (tonkers-kuma) [inline contracts/Joint.sol:180]: hedgeBudget should be 50 here? It's 50 in the init, but 100 here at clone

**Review by tonkers-kuma — COMMENTED** [review]
small initialization thing. Otherwise LG

**Comment by jmonteer** [comment]
old. part of hedgilv2 now

**Pull request closed without merging**